api_tests/src/actors/actor.ts

```diff
@@ -425,6 +425,19 @@ export class Actor {
         await this.assertRedeemed();
     }
 
+    /**
+     * Wait for and execute the refund action
+     */
+    public async refund() {
```

Comment:
📣 use `assertAndExecuteNextAction` instead, do not add this new method.

Reply:
Also, in `hbit_herc20.ts`, do not use `redeem`, etc instead use `assertAndExecuteNextAction`.

Reply:
Oh, yeah. Nice! I'll remove all these actions. We weren't actually using them after the last patch.